index.d.ts

```diff
@@ -1,8 +1,9 @@
 /**
 Create an error from multiple errors.
 */
-declare class AggregateError extends Error implements Iterable<Error> {
-	readonly name: 'AggregateError';
+declare class AggregateError<T extends Error = Error> extends Error
+	implements Iterable<T> {
+	readonly name: "AggregateError";
```

Comment:
Don't do unrelated changes

Reply:
sorry that I missed that, I'll take care of it right away